about.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" type="text/css" href="style/about.css"/>
+    <title>About Firemen Meeting</title>
+</head>
+<body>
+
+  <header>
+    <ul class="logBar">
+      <il class="logBarItem"><img class="logBarImage" src="assets/about/facebook.svg"></il>
+      <il class="logBarItem"><img class="logBarImage" src="assets/about/twitter.svg"></il>
+      <il class="logBarItem">English</il>
+      <il class="logBarItem">My Page</il>
+      <il class="logBarItem">Logout</il>
+    </ul>
+    <div id="topMenu">
+      <img id="menuButton" src="assets/hambMenu.png">
+      <a class="logoHeader" href="./index.html"><img id="logo" src="assets/logo.svg"></a>
+      <nav>
+        <a class="navItems navAbout" href="./about.html">About</a></li>
+        <a class="navItems" href="#mainProgram">Program</a></li>
+        <a class="navItems" href="#join">Join</a></li>
+        <a class="navItems" href="#partner">Sponsor</a></li>
+        <a class="navItems" href="#news">News</a></li>
+        <a class="navItems campaign" href="#CC campaign">CC campaign</a></li>
+      </nav>
+      <ul id="menu">
+        <li id="closeMenuBtn"><a class="closebtn close_menu">&times;</a></li>
+        <li class="menuItems"><a href="./about.html">About</a></li>
+        <li class="menuItems"><a href="#program">Program</a></li>
+        <li class="menuItems"><a href="#join">Join</a></li>
+        <li class="menuItems"><a href="#sponsor">Sponsor</a></li>
+        <li class="menuItems"><a href="#news">News</a></li>
+        <li class="menuItems"><a href="#CC campaign">#CC campaign</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <section id="blurSection">
+
+    <section id="home">
+      <h3>"Hello! Firefighters!"</h3>
+      <h1>FIREMEN MEETING GLOBAL SUMMIT 2023</h1>
```

Comment:
- [ ]  It would be better if you use `<hx>` tag hirarecy properly, in a way that `<h1>` tag should come first then `<h2>`,  `<h3>` and so on ...

You can also use `<p>` tag for "Hello! Firefighters!" text since its just a plain text and not a title or subject to be wrapped in a `heading` tag

The purpose of the `heading` tags is semantics. Correct use of headings on a page provides hierarchy and overview, and helps quickly communicate what a web page is about. They help visitors quickly scan a page, and they help search engines quickly understand a page's structure and what it's about. If you want your heading to be a particular font please use `CSS` properties for that.